### ISSUE TITLE
cmake error, can't build file 

### ISSUE THREAD

**Issue by SunBeenMoon** [foss-for-synopsys-dwc-arc-processors/embarc_mli#309]
I have the build error.

![image](https://user-images.githubusercontent.com/78002830/108703716-57733900-754e-11eb-8bfb-ba019d251735.png)
![image](https://user-images.githubusercontent.com/78002830/108707486-690b0f80-7553-11eb-8bd0-078aeff8ce50.png)

```
gmake all 
gmake -C lib/make TCF_FILE=../../hw/em9d.tcf
gmake[1]: Entering directory 'C:/VSmachine/embarc_mli-mli_dev/lib/make'
cmake \
	-DDEBUG_BUILD=ON \
	-DEXT_CFLAGS= \
	-DOPTMODE=speed \
	-DGEN_EXAMPLES=1 \
	-DMLI_BUILD_REFERENCE=OFF \
	-DCMAKE_INSTALL_PREFIX=C:/VSmachine/embarc_mli-mli_dev/bin/arc \
	-DARC_CFG_TCF_PATH=C:/VSmachine/embarc_mli-mli_dev/hw/em9d.tcf -DCMAKE_TOOLCHAIN_FILE=C:/ARC/MetaWare/arc/cmake/arc-mwdt.toolchain.cmake -G "Unix Makefiles" \
	-BC:/VSmachine/embarc_mli-mli_dev/obj/arc \
	-SC:/VSmachine/embarc_mli-mli_dev
-- The C compiler identification is Clang 10.0.1
-- The CXX compiler identification is Clang 10.0.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/VSmachine/ARC/MetaWare/arc/bin/ccac.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/VSmachine/ARC/MetaWare/arc/bin/ccac.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- MLI_PLATFORM
-- MLI_PLATFORM
-- MLI_PLATFORM
-- Configuring done
-- Generating done
-- Build files have been written to: C:/VSmachine/embarc_mli-mli_dev/obj/arc
cmake --build C:/VSmachine/embarc_mli-mli_dev/obj/arc --target install 
gmake[2]: Entering directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
gmake[3]: Entering directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
gmake[4]: Entering directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
Scanning dependencies of target mli
gmake[4]: Leaving directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
gmake[4]: Entering directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
[  1%] Building CXX object lib/CMakeFiles/mli.dir/src/helpers/src/mli_helpers.cc.o
clang (LLVM option parsing): Unknown command line argument '-arc-scev-aa-vdsp=false'.  Try: 'clang (LLVM option parsing) --help'
clang (LLVM option parsing): Did you mean '--arc-scev-aa=false'?
gmake[4]: *** [lib/CMakeFiles/mli.dir/build.make:82: lib/CMakeFiles/mli.dir/src/helpers/src/mli_helpers.cc.o] Error 1
gmake[4]: Leaving directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
gmake[3]: *** [CMakeFiles/Makefile2:274: lib/CMakeFiles/mli.dir/all] Error 2
gmake[3]: Leaving directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
gmake[2]: *** [Makefile:149: all] Error 2
gmake[2]: Leaving directory 'C:/VSmachine/embarc_mli-mli_dev/obj/arc'
gmake[1]: Leaving directory 'C:/VSmachine/embarc_mli-mli_dev/lib/make'
gmake[1]: *** [C:/VSmachine/embarc_mli-mli_dev/lib/make/../..//make/rules.mk:66: lib] Error 2
gmake: *** [Makefile:17: lib] Error 2
```

developer of this library, do you have solution for this build issue?

**Comment by jcampbel** [comment]
Hi @SunBeenMoon 
It looks like you are trying to use the mli-dev branch. This is a working copy for MLI 2.0 and it does not yet support EM processors.
Please instead take the released version of MLI 1.1, which you can find on the releases page:
https://github.com/foss-for-synopsys-dwc-arc-processors/embarc_mli/releases
MLI 1.1 uses gmake, not Cmake
Best regards,
Jamie

**Comment by SunBeenMoon** [comment]
Thank you for your reply Jamie : - )
I tried MLI1.1, that's work well!
